salt/utils/pkg/deb.py

```diff
@@ -226,7 +813,7 @@ def _get_opts(line):
     """
     Return all opts in [] for a repo line
     """
-    get_opts = re.search(r"\[(.*?=.*?)\]", line)
+    get_opts = re.search(r"\[(.*=.*)\]", line)
```

Comment:
Was this an intentional change to make it non-greedy and why? I assume there was a reason it was non-greedy.